python/ray/tune/result.py

```diff
@@ -25,7 +25,7 @@
 # Number of timesteps in this iteration.
 TIMESTEPS_THIS_ITER = "timesteps_this_iter"
 
-# (Optional/Auto-filled) Accumulated time in seconds for this experiment.
+# (Auto-filled) Accumulated number of timesteps for this entire experiment.
```

Comment:
Thanks for the catch! Can you put the `Optional` part back in? Users can override this field.

Reply:
Oh yes, sure! I thought that was also obsolete.

Reply:
Oh my page wasn't refreshed... now seeing @ericl's comment.